src/modules/import/file.js

```diff
@@ -10,7 +10,7 @@ goog.provide('ngeo.File');
  * @ngInject
  */
 ngeo.File = function($q, $http, $window, gettext) {
-  var fileReader, canceler;
+  let fileReader, canceler;
```

Comment:
@sbrunner tested it: the closure comments were preserved and the ES6 constructs like `for of` where replaced.